code/modules/mining/ores_coins.dm

```diff
@@ -564,3 +564,29 @@
 		to_chat(user, "<span class='notice'>You detach the string from the coin.</span>")
 	else
 		..()
+
+///////////////////////////////////////////////////////////
+
+/obj/item/weapon/coin/pomf
+	material="pomf"
+	name = "pomf coin"
+	desc = "A platinum coin featuring the effigy of a white chicken. Few know of its true value. Fewer still can make use of it."
+	icon_state = "coin_pomf"
+	credits = 2525
+	siemens_coefficient = 1
+	melt_temperature=1768+T0C
+	force = 4
+	throwforce = 4
+
+/obj/item/weapon/coin/pumf
+	material="pumf"
+	name = "pumf coin"
+	desc = "A slade coin featuring the effigy of an angry chicken. If you come into its possession that means you've been a naughty boy, whatever you've been doing stop it."
```

Comment:
`"if it comes into your possession"` is proper in English
Also, the back end of this sentence needs to be separated as it is an independent phrase:
`"...you've been a naughty boy. Whatever you've been doing, stop it."`